Cargo.toml

```diff
@@ -150,6 +150,7 @@ alloy-primitives = "0.5"
 alloy-dyn-abi = "0.5"
 alloy-sol-types = "0.5"
 alloy-rlp = "0.3"
+alloy-chains = "0.1.7"
```

Comment:
this needs features:

https://github.com/alloy-rs/chains/blob/a36d918234a40e62f932507448c8c3e1e040be16/src/chain.rs#L110-L175

Reply:
okay got it